Basic Background Generator/Readme.md

```diff
@@ -0,0 +1,33 @@
+# BASIC BACKGROUND GENERATOR
+
+## A basic gradient Background Generator which allows you to create a background by selecting your required colors. This will make up a gradient mix and provide you the code line so that you can use it in your Website 
+
+<br>
+
+## Tech Stack Used
+<br>
+
+![HTML](https://img.shields.io/badge/html5%20-%23E34F26.svg?&style=for-the-badge&logo=html5&logoColor=white)
+![CSS](https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white)
+![JS](https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E)
+
+## How to use:
+<br>
+- Download or clone the repository
+
+```
+git clone https://github.com/Ayushparikh-code/Web-dev-mini-projects.git
+```
+- Go to the directory
+- Run the index.html file in your browser
+- Start creating your own gradient background
+<br>
+## Use of the Project
+
+This project aims to find the best looking background gradient for your project. You just have to select the colors you want from the color picker and then copy the code displayed and paste into your code as background color.
+It's That Simple!!
+<br><br>
+
+![image](https://user-images.githubusercontent.com/67221487/124392353-ab6b0700-dd12-11eb-9093-b6309b08e2d5.png)
+<br>
+![image](https://user-images.githubusercontent.com/67221487/124392372-c2115e00-dd12-11eb-9743-9be785121fb4.png)
```

Comment:
Use Markdown beautifier for the complete file.
Here use this link :
https://codebeautify.org/markdown-formatter

Reply:
Changes made, please let me know if any more are required!
@chicken-biryani